### PR TITLE
Update hello-cert-trust-policy-settings.md

### PR DESCRIPTION
**Pull request by nenonix**
lines 38-42 edited for clarity and brevity - closes https://github.com/MicrosoftDocs/windows-itpro-docs/issues/2064

### CODE REVIEW

**Comment by mypil** [comment]
@DaniHalfin - can you please assist to check, approve and merge the content for this PR based on issue #2064?

Thank you.

**Comment by DaniHalfin** [comment]
A couple of notes on this PR - 
The way it is described now, there are 4 main points to make : 
1. This Group Policy setting detrmines whether users are allowed and prompted to enroll for Windows Hello for Business.
2. It can be configured for both computers and users.
3. When configured for computers, all users that sign-in to those computers will be allowed and prompted to enroll for Windows Hello for Business.
4. When configured for users, only those users will be allowed and prompted to enroll for Windows Hello for Business.

I don't see a need to talk about precedence, as this is in accordance of group policy processing (not a special case), so I would keep that out.

With that in mind, I'd like it to read as "active" talk (and not passive). An example from the original description would be : "You can configure...". We'd like to address our reader and help them get to where they are going, so we direct our content at them, and also try to describe goals and actions before we describe details.

I'd like to see the above reflected here, if possible. If not, I'd try to find time in the next couple of weeks to make the edit myself.

Small side note : it should be referred to as enrollment (and not auto-enrollment). Unless I'm getting something wrong.

Adding @mapalko (just FYI).

**Comment by mypil** [comment]
@DaniHalfin - Thanks for your immediate response.

@nenonix  - Please take note of the recommendation and kindly make the necessary changes on this PR. Thank you.

**Comment by DaniHalfin** [comment]
Also, I hope I'm not wrong about the prompt detail. @mapalko can confirm.

**Comment by DaniHalfin** [comment]
btw the quick fix here would be : 
`The **Enable Windows Hello for Business** Group Policy setting is the configuration needed for Windows to determine if a user should attempt to enroll for Windows Hello for Business. A user will only attempt enrollment if this policy setting is set to **enabled**.`

I don't disagree that this needs a rewrite, so I'm very open to that. However, if a rewrite is too much, we can just implement the above fix - removed one word, changed another, and added formatting for clarity.

**Comment by nenonix** [comment]
I will wait for confirmation and then work on that rewrite. Thanks

**Comment by mypil** [comment]
@mapalko - Can you possibly confirm if the content of this PR is correct and can be merged? Thank you.

**Comment by nenonix** [comment]
@mypil I am still working on this. I will attempt the rewrite and if that doesn't do it we can use the quick fix.

**Comment by DaniHalfin** [comment]
Started a review on one part of the commit. @nenonix - whenever you can please take a look.

**Comment by nenonix** [comment]
@DaniHalfin I am not seeing your review. Am I missing something?

File: windows/security/identity-protection/hello-for-business/hello-cert-trust-policy-settings.md [hunk 1]
```diff
@@ -35,9 +35,9 @@ On-premises certificate-based deployments of Windows Hello for Business needs th
 
 ## Enable Windows Hello for Business Group Policy
 
-The Enable Windows Hello for Business Group Policy setting is the configuration needed for Windows to determine if a user should be attempt to enroll for Windows Hello for Business.  A user will only attempt enrollment if this policy setting is configured to enabled.  
+The Group Policy setting determines whether users are allowed, and prompted, to enroll for Windows Hello for Business. You must first enable the Windows Hello for Business Group Policy, then you can configure it for computers or users.
```
[the remark this answers is not in the document]
Comment (DaniHalfin) [reply windows/security/identity-protection/hello-for-business/hello-cert-trust-policy-settings.md]: Forgot to post. Sorry about that.

**Comment by nenonix** [comment]
@Justinha done.( I am using this as an example for our team to capture the copy review process. so please bear with us
)

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/identity-protection/hello-for-business/hello-cert-trust-policy-settings.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ On-premises certificate-based deployments of Windows Hello for Business needs th
 
 ## Enable Windows Hello for Business Group Policy
 
-The Group Policy setting determines whether users are allowed, and prompted, to enroll for Windows Hello for Business. You must first enable the Windows Hello for Business Group Policy, then you can configure it for computers or users.
+The Group Policy setting determines whether users are allowed, and prompted, to enroll for Windows Hello for Business. It can be configured for computers or users.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-cert-trust-policy-settings.md]: Cpoy edited. No changes.

Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-cert-trust-policy-settings.md]: Copy edited - no changes.

**Comment by DaniHalfin** [comment]
Looks good.

**Pull request merged**